test/test_helper.rb

```diff
@@ -20,18 +20,29 @@
 module KubernetesDeploy
   class TestCase < ::Minitest::Test
     def setup
+      ColorizedString.disable_colorization = true unless ENV["PRINT_LOGS"]
       @logger_stream = StringIO.new
-      @logger = ::Logger.new(@logger_stream)
-      @logger.level = ::Logger::INFO
-      KubernetesDeploy.logger = @logger
-      KubernetesResource.logger = @logger
+    end
+
+    def test_logger
+      @test_logger ||= begin
+        device = ENV["PRINT_LOGS"] ? $stderr : @logger_stream
+        KubernetesDeploy::Logger.build(@namespace, KubeclientHelper::MINIKUBE_CONTEXT, device)
+      end
     end
 
     def teardown
+      ColorizedString.disable_colorization = false
       @logger_stream.close
     end
 
     def assert_logs_match(regexp, times = nil)
+      if ENV["PRINT_LOGS"]
```

Comment:
Do you want to keep `PRINT_LOGS` in the test suite for debugging? I'm not against it, actually I used something like this myself too.